### ISSUE TITLE
Push `@tableland/sdk` to skypack to fix import error

### ISSUE THREAD

**Issue by dtbuchholz** [tablelandnetwork/js-tableland#80]
## Expected Behavior
I should be able to use `@tableland/sdk` in skypack and properly import `connect`.

However, Tableland doesn't appear on [skypack](https://www.skypack.dev/search?q=%40tableland%2Fsdk&p=1) (here's the skypack cdn link, for reference: [here](https://cdn.skypack.dev/@tableland/sdk)). Also, an error is thrown in the browser's console:
```
Uncaught SyntaxError: The requested module '/-/@tableland/evm@v2.0.2-eEo31D2jP28UYuDJXFJx/dist=es2019,mode=imports/unoptimized/proxies.js' does not provide an export named 'proxies' (at connector-1be2aba6.js:3:9)
```

## Steps to Reproduce the Problem
1. Go to https://codepen.io/dtbuchholz/pen/RwQdwoe
2. Open inspector/console
3. Click `Connect` button to see error

## Specifications

- Version: https://cdn.skypack.dev/@tableland/sdk
- Platform: Mac / Brave browser

**Comment by carsonfarmer** [comment]
Requires: https://github.com/tablelandnetwork/evm-tableland/pull/71

**Comment by wbt** [comment]
This should be closed as not relevant to js-tableland per the logic in [this issue](https://github.com/tablelandnetwork/js-tableland/issues/87).

**Comment by sanderpick** [comment]
looks like https://github.com/tablelandnetwork/evm-tableland/pull/71 is merged. @carsonfarmer do we need to do anything else to resolve this issue?

**Comment by carsonfarmer** [comment]
> This should be closed as not relevant to js-tableland per the logic in [this issue](https://github.com/tablelandnetwork/js-tableland/issues/87).

I'm not sure I follow this comment? `evm-tableland` is a dependency of _this library_, and in order for _this library_ to be usable with skypack, that work needs to be merged _first_. Once it _has_ been merged (which is now the case), a new release of this library (now pushed) will make skypack a viable option.

**Comment by carsonfarmer** [comment]
> looks like [tablelandnetwork/evm-tableland#71](https://github.com/tablelandnetwork/evm-tableland/pull/71) is merged. @carsonfarmer do we need to do anything else to resolve this issue?

This issue has been resolved with the most recent release. We can close this as "done" I think. Will confirm shortly.

**Comment by carsonfarmer** [comment]
This should be fixed with [8353427](https://github.com/tablelandnetwork/js-tableland/commit/835342751a10ae9bb7f93a9618bee36ae3daadcc)
